templates/vue/src/components/Lightbox/media/MultiContentMedia/PageRows.vue

```diff
@@ -5,82 +5,98 @@
+                  <i
+                    class="fas fa-heart fa-sm"
+                    :style="{
+                      color: isFavourite(row.node.id) ? 'red' : 'black',
+                      opacity: isFavourite(row.node.id) ? '1' : '0.25',
+                      cursor: 'pointer',
+                    }"
+                    @click="toggleFavourite(row.node.id)"
+                  ></i>
+                </a>
+              </div>
+              <div v-if="disableRow(index, row.node)">
+                <h1 class="title">
+                  {{ row.node.title }}
+                </h1>
+                <locked-content
+                  :node="row.node"
+                  :condition-node="index > 0 ? rows[index - 1].node : {}"
+                ></locked-content>
+              </div>
+              <div v-else :data-qa="`row-content-${row.node.id}`">
+                <div v-if="row.node.mediaType !== 'multi-content'">
+                  <tapestry-media
+                    :node-id="row.node.id"
+                    :dimensions="dimensions"
+                    context="page"
+                    style="margin-bottom: 24px;"
+                    @complete="updateProgress(row.node.id)"
+                    @load="handleLoad($refs.rowRefs[index])"
+                  />
+                  <p v-if="row.children.length > 0 && !areAllPopup(row.children)">
+                    {{ row.node.typeData.subAccordionText }}
+                  </p>
+                  <accordion-rows
+                    v-if="row.children.length > 0"
+                    :dimensions="dimensions"
+                    :node="row.node"
+                    :rowId="subRowId"
+                    context="page"
+                    :level="level + 1"
+                    @changeRow="changeRow"
+                    @load="handleLoad"
+                    @updateProgress="updateProgress"
+                  ></accordion-rows>
+                </div>
+                <multi-content-media
+                  v-else-if="row.children.length > 0"
+                  :node="getNode(row.node.id)"
+                  :row-id="subRowId"
+                  context="page"
+                  :level="level + 1"
+                  @close="handleAutoClose"
+                  @complete="updateProgress"
+                />
+              </div>
+              <button
+                v-if="row.node.completed && isVisible(row)"
+                class="mt-2"
+                @click="hasNext ? next() : (showCompletion = true)"
+              >
+                {{ node.typeData.finishButtonText }}
+              </button>
+            </b-col>
+            <b-col
+              v-if="insertInstructions[index].includes('col')"
+              :key="'column' + row.node.id"
+              class="w-50"
+            ></b-col>
+            <div
+              v-if="insertInstructions[index].includes('spacer')"
+              :key="'spacer' + row.node.id"
+              class="w-100"
+            ></div>

```

Comment:
Oh, I think you need to familiarize yourself with the [Bootstrap grid system](https://getbootstrap.com/docs/4.0/layout/grid/) (also look at the [Bootstrap Vue version](https://bootstrap-vue.org/docs/components/layout#columns-b-col)). There is a much simpler way of doing this that doesn't require inserting empty columns or spacers.

Reply:
@EldadZZipori I have made these changes now.